functional_tests.go

```diff
@@ -6537,7 +6538,7 @@ func main() {
 		testComposeObjectErrorCases()
 		testCompose10KSources()
 		testUserMetadataCopying()
-		testEncryptedCopyObject()
+		// testEncryptedCopyObject()
```

Comment:
testEncryptedCopyObjectV2 testEncryptedCopyObject commented out, intentional?

Reply:
Yes CopyObject is not supported yet with SSE-C commenting it out opening a new issue to re-enable it with server feature in place.

Reply:
As per discussion, we will return "Not Implemented" from the server instead of commenting this test out here.